[Pages] redirect clarification (#3784)
cloudflare/cloudflare-docs: content/pages/how-to/use-bulk-redirects.md

The redirects defined in the `_redirects` file of your build folder can work together with your Bulk Redirects. In case of duplicates, Bulk Redirects will run in front of your Pages project, where your other redirects live.

> For example, if you have Bulk Redirects set up to direct `abc.com` to `xyz.com` but also have `_redirects` set up to direct `xyz.com` to `foo.com`, the request will resolve to `foo.com`.

Reviewer: ```suggestion
For example, if you have Bulk Redirects set up to direct `abc.com` to `xyz.com` but also have `_redirects` set up to direct `xyz.com` to `foo.com`, a request for `abc.com` will eventually redirect to `foo.com`.
```

Reviewer: "Resolve" makes it sound like we're flattening it, which we we're not. The client will be served a redirect to `xyz.com`. A subsequent request to `xyz.com` will then serve another redirect to `foo.com`.
